pkg/client/client.go

```diff
@@ -61,6 +61,9 @@ func (c *Client) CreateDataVault(config *models.DataVaultConfiguration) (string,
 		return "", fmt.Errorf("failed to marshal data vault configuration: %w", err)
 	}
 
+	logger.Debugf("Sending request to create a new data vault with the following data vault configuration: %s",
```

Comment:
what's the point of displaying raw bytes??

Reply:
It will actually display as a string, not like [101,64,52]. Or did you mean something else?